pyaedt/modeler/cad/Primitives.py

```diff
@@ -592,11 +592,10 @@ def does_object_exists(self, object):
             else:
                 return False
         else:
```

Comment:
Here the entire code:
```
        if isinstance(object, int):
            if object in self.objects:
                return True
            else:
                return False
        else:
            if object in self.objects_by_name:
                return True
            else:
                return False
```

Could become:
  ```
      if isinstance(object, int):
            if object in self.objects:
                return True
            elif object in self.objects_by_name:
                return True
            return False
```

Reply:
thanks. Improved